src/app/submit.service.ts

```diff
@@ -1,14 +1,14 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { Observable, of } from 'rxjs';
-import { mergeMap, catchError } from 'rxjs/operators';
+import { mergeMap, catchError, tap, delay } from 'rxjs/operators';
 import { OpenBabelData } from './OpenBabelData';
 
 @Injectable({
   providedIn: 'root'
 })
 export class SubmitService {
-  private readonly submitUrl: string = "/submit/";
+  private readonly submitUrl: string = "/submit/test";
```

Comment:
test is not appropriate route name.  perhaps you need to rename the route to be 'api/submit'